Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -311,6 +311,8 @@ private void SetLocatorsGridView()
             defView.GridColsView = new ObservableList<GridColView>();
             defView.GridColsView.Add(new GridColView() { Field = nameof(ElementLocator.Active), WidthWeight = 8, MaxWidth = 50, HorizontalAlignment = System.Windows.HorizontalAlignment.Center, StyleType = GridColView.eGridColStyleType.CheckBox });
             List<GingerCore.General.ComboEnumItem> locateByList = GingerCore.General.GetEnumValuesForCombo(typeof(eLocateBy));
+            locateByList.Remove(locateByList.Where(x=>((eLocateBy)x.Value) == eLocateBy.POMElement).FirstOrDefault());
```

Comment:
if .FirstOrDefault() will return null so then you code will try to Remove(null) and exception will throw, need to check first if not null and then do Remove